packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,58 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import Logger from 'ember-metal/logger';
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor(assert) {
+    super();
+
+    this.assert = QUnit.assert;
```

Comment:
@rwjblue informed me that `QUnit.Assert` is not what we want here. Apparently we should be doing `this.assert = QUnit.config.current.assert` instead.

Also, I think I confused you. I was saying that we can do this line in the constructor in the super super super super class (`TestCase`) since it just makes sense to do it for everyone.

As you pointed out, there is a `let assert = QUnit.assert;` in the `abstract-test-case` file, which would also be wrong according to Robert. However, if we have `this.assert` available all the time, we can just update the other helpers in that file to use that instead.

Does that make sense?